apps/Standalone/src/mcp/app/McpStandard.tsx

```diff
@@ -70,66 +86,78 @@ const AwaitingConnection = () => {
 
 export const McpStandard = () => {
   const styles = useMcpStandardStyles();
-  const { theme, appId, hostingPlan } = useSelector((state: RootState) => ({
+  const { theme, appId } = useSelector((state: RootState) => ({
     theme: state.workflowLoader.theme,
     appId: state.workflowLoader.appId,
-    hostingPlan: state.workflowLoader.hostingPlan,
   }));
 
+  const hostingPlan = 'standard';
+  const [shouldReload, setShouldReload] = useState<boolean | undefined>(undefined);
   const hasValidAppId = !!appId && appId.trim() !== '';
   const resourceIdValidation =
     /^\/subscriptions\/[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}\/resourceGroups\/[a-zA-Z0-9](?:[a-zA-Z0-9-_.]*[a-zA-Z0-9])?\/providers\/[a-zA-Z0-9-_.]+\/[a-zA-Z0-9-_./]+$/;
   const isValidResourceId = hasValidAppId && resourceIdValidation.test(appId);
 
+  const resourceDetails = {
+    subscriptionId: 'f34b22a3-2202-4fb1-b040-1332bd928c84',
+    resourceGroup: 'TestACSRG',
+    location: 'westus',
+  };
   const {
     data: workflowAppData,
     isLoading: isWorkflowLoading,
     error: workflowError,
-  } = useWorkflowApp(isValidResourceId ? appId : '', hostingPlan, isValidResourceId);
+  } = useWorkflowApp(isValidResourceId ? appId : '', hostingPlan, isValidResourceId && !!appId);
```

Comment:
nit: check for !!appId is redundant

Reply:
Will remove it in my next PR.... was not sure during merge with main, so just kept it there